templates/error.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <title>Build error</title>
+    <script src="/ember-cli-live-reload.js" type="text/javascript"></script>
```

Comment:
Forking the middleware sounds good, and then we pull stuff back in once things have stabilized in Broccoli core.